internal/config/network.go

```diff
@@ -37,6 +40,9 @@ var MainNet = Network{
 	// DCP0010Height on mainnet is block
 	// 00000000000000002f4c6aaf0e9cb4d5a74c238d9bf8b8909e2372776c7c214c.
 	DCP0010Height: 657280,
+	// DCP0012Height on mainnet is block
+	// 071683030010299ab13f139df59dc98d637957b766e47f8da6dd5ac762f1e8c7.
```

Comment:
Confirmed.

```sh
$ dcrctl getblockchaininfo | jq -c '.deployments.changesubsidysplitr2 | {status,since}'
{"status":"active","since":794368}

$ dcrctl getblockchaininfo | jq '.deployments.changesubsidysplitr2.since' | dcrctl getblockhash -
071683030010299ab13f139df59dc98d637957b766e47f8da6dd5ac762f1e8c7
```